content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -212,3 +213,11 @@ model Device {
   data Unsupported("circle")
 }
 ```
+
+## Prepared statement caching
+
+A [prepared statement](https://www.postgresql.org/docs/current/sql-prepare.html) is a feature that can be used to optimize performance. A prepared statement is parsed, compiled, and optimized only once and then can be executed directly multiple times without the overhead of parsing the query again. This way, Prisma Client's [query engine](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine) does not repeatedly prepare the same statement but can reuse an existing one which reduces database CPU usage and query latency.
```

Comment:
Prisma's behaviour is currently not super clear from this.

> This way, Prisma Client's [query engine](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine) does not repeatedly prepare the same statement but can reuse an existing one which reduces database CPU usage and query latency.

What does "this way" refer to exactly here? The two previous sentences were about "prepared statements" but it doesnt't say _how_ Prisma utilizes prepared statements exactly. It also doesn't state whether the dveloper can in any way influence this behaviour.

Reply:
Clarified. Please have another look @nikolasburk